doc/developer/alembic_on_custom_schema.rst

```diff
@@ -0,0 +1,103 @@
+.. _alembic_on_custom_tables:
+
+Alembic on custom tables
+========================
+
+If you want to manage custom tables (GMF tables; not in the ``main`` nor the ``static`` schema)
+with an alembic you must setup a custom alembic process. Here's an example. Others solutions
+are possible, but do not try to reuse the existing ``alembic.ini``.
+
+The following example use ``docker cp``. You can achieve the same with a volume, but you will
+have to set the owner (root) of copied files.
+
+Init alembic
+------------
+
+In your project, add an ``geoportal/alembic_<project_name>`` folder.
+
+Build and run you project.
+
+In the running container initialize alembic in a folder that have **not** already
```

Comment:
In the running container, initialize alembic in a folder that does **not** already have